tests/basic/distribute/file-create.t

```diff
@@ -23,8 +23,8 @@ TEST pidof glusterd
 # We want fixed size bricks to test min-free-disk
 
 # Create 2 loop devices, one per brick.
-TEST   truncate -s 25M $B0/brick1
-TEST   truncate -s 25M $B0/brick2
+TEST   truncate -s 300M $B0/brick1
+TEST   truncate -s 300M $B0/brick2
```

Comment:
Line 64 in this test needs to fill the brick to more than 40% of the brick, this test fails on my local setup even after the change.